components/feature/logins/src/androidTest/java/mozilla/components/feature/logins/LoginExceptionStorageTest.kt

```diff
@@ -37,6 +37,7 @@ class LoginExceptionStorageTest {
     var instantTaskExecutorRule = InstantTaskExecutorRule()
 
     @get:Rule
+    @Suppress("DEPRECATION")
```

Comment:
what's throwing the "deprecation" warning here?

Reply:
Left a note in the commit message. The constructor used there will become deprecated following the `room` version update. To be handled in https://github.com/mozilla-mobile/android-components/issues/11765.